docs/docs/config-file.md

```diff
@@ -0,0 +1,46 @@
+## Paths
+
+The default xemu configuration file can be found at the following locations after first run:
+
+| OS | Path |
+|----|------|
+| Linux | `~/.local/share/xemu/xemu/xemu.ini` |
+| macOS |  |
```

Comment:
macOS: /Users/username/Library/Application\ Support/xemu/xemu/xemu.ini
Windows: C:\Users\username\AppData\Roaming\xemu\xemu\xemu.ini

Should note that xemu.ini can be created in xemu executable directory, or explicitly given via `-config_path` command line argument.